### DISCUSSION TITLE
Do you accept an only partially translated I8N as MR ?

### DISCUSSION THREAD

**Discussion by lumpidu** [zitadel/zitadel#6933] [Q&A]
I have seen, there are multiple files with different target audiences for the I8N parts. We would like to just translate the client facing part of it to Icelandic (EMail, Login Interface) and leave the rest to English, because this would mean otherwise a tremendous work for us. OTOH keeping this translation around in an own version of Zitadel would also be tedious.

So basically my question is: would you accept a partial I8N solution as contribution as well ?

**Comment by hifabienne** [comment]
Hei @lumpidu 
Thanks for the question.
We do have some discussions around that topic. I think at the moment it is not possible from a technical perspective, that you can only use the translations for the login/register ui, but not in the console. As the console would then also search for the icelandic tranlsations and not tranlsate in the default language. 

Of course we would accept the translation for only the user facing parts, as soon as ZITADEL is capable in handling those problems. 
It relates to this issue: https://github.com/zitadel/zitadel/issues/4393
As well as to this one: https://github.com/zitadel/zitadel/issues/6560
As we thought it might be the easiest to first look at how we handle the tranlsations in general, starting with the error messages and then continue with other parts.

**Reply by lumpidu** [reply]
In general in Iceland if there is a missing I8N, people switch to English. A partial translation would already be better, because users could already set their locale to Icelandic in the signup and read most relevant user facing messages in Icelandic as well, whereas untranslated entries would still be in English. Technically that should be possible by providing Icelandic texts for the user-facing parts and just fill in the English localization for the other parts that are not translated (yet).
